role.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 12:16:52 2016
+
+@author: mina
+"""
+
+class Role():
```

Comment:
Similar to my comment for the Variant object, an abstract Role object would have an identifier, a description, a boolean required value and a boolean ha value.